delta/plugins/blazegraph/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/routes/BlazegraphViewsQuerySpec.scala

```diff
@@ -297,6 +299,19 @@ class BlazegraphViewsQuerySpec(docker: BlazegraphDocker)
         )
       )
     }
+
+    "have a correct details when the query is malformed" in {
+      val proj                 = view1Proj1.project
+      val sparqlQueryWithError = SparqlQuery("SELEC")
+      val result               = viewsQuery.query(view1Proj1.viewId, proj, sparqlQueryWithError, SparqlNTriples)
+
+      rejectionDetailsOf(result) should contain(
+        "org.openrdf.query.MalformedQueryException: Lexical error at line 1, column 6.  Encountered: <EOF> after : \"SELEC\""
```

Comment:
How does it behave for a larger query ?
Is it still readable ?

Reply:
Even when the query is large, this line is pretty concise and points only to the error itself. If you have a query in mind I can test that still